client/app/domain/domain.controller.js

```diff
@@ -192,6 +194,10 @@ angular.module('App').controller(
         .finally(() => {
           this.loading.domainInfos = false;
           this.$scope.$broadcast('domain.refreshData.done');
+
+          if (!this.domain.isExpired) {
+            this.$state.go(`${this.getState()}.information`, this.$stateParams);
```

Comment:
`return` ?

Reply:
As we're in the `finally` hook, the value will never be used (when chaining we'll get the result from the `then`/`catch` hook )